explainer.md

```diff
@@ -1489,19 +1489,22 @@ const settingsBucket = await navigator.storageBuckets.openOrCreate("settings", {
 const emailsPerPage = settingsBucket.localStorage.getItem('emailsPerPage');
 ```
 
-This alternative was rejected because we were concerned about the performance
-implications of supporting multiple `localStorage` instances per origin.
-
-Due to the synchronous nature of the Web Storage API, user agents implement
-`localStorage` via a full in-memory cache.
-
-TODO: Explain why we chose not to integrate `localStorage`.
-
-* LocalStorage doesn't follow durability policies. Sync write API means we need
-  in-renderer caches, so we couldn't support either `"relaxed"` or `"strict"`
-* RAM consumption limitation. Would need special quota system just for
-  LocalStorage. Buckets with small `quota` would get access then.
-* Non-issue: sync read API. We can read all data on `openOrCreate()`.
+This alternative was rejected because of `localStorage` does not follow
+the same durability policies, and we would need to create a quota system
+just to support it.
+
+Due to the synchronous nature of Web Storage API, user agents would need an
+in-renderer cache to support writes for `localStorage`.
+Therefore we would be able to support `"relaxed"` or `"strict"` policies.
+
+We would also need a special quota system just for `localStorage` and
+its RAM consumption. This could have performance implications of supporting multiple
+`localStorage` instances per origin. Each bucket could have a smaller quota than
+the existing quota limit but would add extra complexity to the implementation. 
+
+At first pass, the synchronous read API of `localStorage` seems like it could be
+a problem. However in this case, buckets could help from its main thread blocking
```

Comment:
"buckets could help" feels like it has the wrong subject.

In this case, user agents don't need to block the main thread to read the `localStorage` contents. Instead, implementations can read the `localStorage` data asynchronously, while processing `navigator.storageBuckets.openOrCreate()`.